hudi-client/hudi-flink-client/src/test/java/org/apache/hudi/index/bloom/TestFlinkHoodieBloomIndex.java

```diff
@@ -133,7 +137,7 @@ public void testLoadInvolvedFiles(boolean rangePruning, boolean treeFiltering, b
         new HoodieAvroRecord(new HoodieKey(rowChange4.getRowKey(), rowChange4.getPartitionPath()), rowChange4);
 
     List<String> partitions = asList("2016/01/21", "2016/04/01", "2015/03/12");
-    List<Pair<String, BloomIndexFileInfo>> filesList = index.loadColumnRangesFromFiles(partitions, context, hoodieTable);
+    List<Pair<String, BloomIndexFileInfo>> filesList = index.loadColumnRangesFromFiles(context.parallelize(partitions, 1), context, hoodieTable);
```

Comment:
why parallelize w/ just 1?
what incase we have 10k physical partitions?

Reply:
We have 3 partitions, should I make it 3?

Reply:
oh, this is test code. sorry. it should be fine.